pkg/login/interactive_login.go

```diff
@@ -27,6 +27,13 @@ func InteractiveLogin(config *config.Config) error {
 
 	config.Profile.DeviceName = getConfigureDeviceName(os.Stdin)
 	config.Profile.TestModeAPIKey = apiKey
+	displayName, err := getDisplayName(nil, stripe.DefaultAPIBaseURL, apiKey)
+
+	if err != nil {
+		displayName = ""
+	}
+
+	config.Profile.DisplayName = displayName
```

Comment:
`getDisplayName` returns `"", err` when there is an error, so we don't need lines 32-34 which get overwritten by line 36 anyway.

```go
displayName, _ := getDisplayName(nil, stripe.DefaultAPIBaseURL, apiKey)
config.Profile.DisplayName = displayName
```
should be enough